analysers/analyser_merge_power_plant_FR.py

```diff
@@ -20,12 +20,10 @@
 ##                                                                       ##
 ###########################################################################
 
-import re
 from io import open # In python3 only, this import is not required
 from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
 from .Analyser_Merge_Geocode_Addok_CSV import Geocode_Addok_CSV
-from .modules import downloader
-from .modules import Stablehash
+from modules import Stablehash
```

Comment:
Please, keep relative import.

Reply:
This is inside analysers. Should then be `..modules` I think. Other locations always import from `modules` by absolute path. For example in Analyser_Merge_Mapillary or analyser_sax. So this sounded more common.

Reply:
ok